python/tvm/rpc/server.py

```diff
@@ -119,6 +119,15 @@ def download_linked_module(file_name):
     return temp
 
 
+def _serve_loop(sock, load_library, work_path=None):
+    """Server loop"""
```

Comment:
del the meaningless comment because it  give no more information beyond the function name, or write a meaning one.